tests/deseq2/test_collapse.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import numpy as np
+
+from inmoose.deseq2 import makeExampleDESeqDataSet, collapseReplicates
+
+class Test(unittest.TestCase):
+
+    def test_collapse(self):
```

Comment:
Can you add a small description as you did for the other functions please

Reply:
👍